modules/contracts/adapters/cw-staking/src/lib.rs

```diff
@@ -25,14 +25,14 @@ pub use msg::StakingQueryMsgFns;
 
 #[cfg(feature = "interface")]
 pub mod interface {
+    use abstract_adapter::abstract_interface::{
```

Comment:
Ahhh we can't re-expose this as `abstract_adapter::interface`? because that's a macro right?

Reply:
No it's because interface is a module inside abstract adapter already

Reply:
I don't see that? `abstract_adapter::interface` doesn't exist yet. Anyway I don't think it's a good idea to re-export it that way anyway because it's not very intuitive.